extensions/resteasy-common/deployment-spi/src/main/java/io/quarkus/resteasy/common/deployment/EndpointValidationPredicatesBuildItem.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.resteasy.common.deployment;
+
+import java.util.function.Predicate;
+
+import org.jboss.jandex.ClassInfo;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class EndpointValidationPredicatesBuildItem extends MultiBuildItem {
```

Comment:
Can you add some Javadoc to indicate what it is?

Build items added to spi modules are considered public API.